src/pages/[platform]/build-a-backend/q-developer/index.mdx

```diff
@@ -90,3 +90,36 @@ import { type ClientSchema, a, defineData } from "@aws-amplify/backend";
 </Callout>
 
 **Step 4:** Make any required changes to the schema and save the `amplify/data/resource.ts` file. This will trigger a sandbox deployment and your new data model will be deployed
+
+## Use Q Developer - workspace in your Amplify project
+
+
+Adding `@workspace` to your question in Amazon Q automatically incorporates the most relevant parts of your workspace code as context, using an index that updates periodically. For more information on `@workspace` functionality, please visit [Amazon Q Developer - Workspace Context] (https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/workspace-context.html). 
```

Comment:
Need to remove space here causing link not to render correctly
<img width="974" alt="image" src="https://github.com/user-attachments/assets/a7d5fb59-8399-443a-9ca0-1fe8d7a921a0">


```suggestion
Adding `@workspace` to your question in Amazon Q automatically incorporates the most relevant parts of your workspace code as context, using an index that updates periodically. For more information on `@workspace` functionality, please visit [Amazon Q Developer - Workspace Context](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/workspace-context.html). 
```